app/models/competency.rb

```diff
@@ -0,0 +1,14 @@
+class Competency < ActiveRecord::Base
+  # Relationships
+  # has_many :indicators
```

Comment:
Yea im basically keeping it there for future use of it when I create the indicator model